examples/api/lib/cupertino/text_field/cupertino_text_field.0.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// To control the text that is displayed in the text field, use the
```

Comment:
I know, it's tempting to put the description text here, but we don't want to do that because then we have the description in two places (once in the API comments, and once here), and they will get out of sync.

Just keep the description in the API comments in the original source file.